tests/components/system_bridge/test_config_flow.py

```diff
@@ -97,78 +115,125 @@ async def test_user_flow(
     assert result["type"] == data_entry_flow.RESULT_TYPE_FORM
     assert result["errors"] is None
 
-    aioclient_mock.get(
-        f"{FIXTURE_BASE_URL}/information",
-        headers={"Content-Type": "application/json"},
-        json=FIXTURE_INFORMATION,
-    )
-
-    with patch(
+    with patch("systembridgeconnector.websocket_client.WebSocketClient.connect"), patch(
```

Comment:
Probably patch the name in the config flow.

https://docs.python.org/3/library/unittest.mock.html#where-to-patch

```suggestion
    with patch("homeassistant.components.system_bridge.config_flow.WebSocketClient.connect"), patch(
```